src/content/docs/en/guides/styling.mdx

```diff
@@ -148,6 +148,15 @@ This pattern lets you style child components directly. Astro will pass the paren
 Because the `class` prop includes the child in its parent’s scope, it is possible for styles to cascade from parent to child. To avoid this having unintended side effects, ensure you use unique class names in the child component.
 :::
 
+### Inline styles
+
+You can style HTML elements inline using the `style` attribute. This can be a CSS string or an object of CSS properties:
+
+```astro title="src/pages/index.astro"
+// These are equivalent:
+<p style={{ color: "brown", textDecoration: "underline" }}>My text</p>
+<p style="color: brown; text-decoration: underline">My text</p>
```

Comment:
```suggestion
<p style="color: brown; text-decoration: underline;">My text</p>
```
I have almost always seen a semi-colon used, even in the final property. I think I might prefer to show this. Thoughts?